<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>815</width>
    <height>923</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTableWidget" name="tableWidget_massive">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>211</width>
      <height>831</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string/>
     </property>
    </column>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>90</y>
      <width>561</width>
      <height>151</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="pushButton_random">
       <property name="text">
        <string>Рандомное количество ячеек</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_removelast">
       <property name="text">
        <string>Удалить последнюю Ячейку</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_randomValues">
       <property name="text">
        <string>Рандомное заполнение ячеек</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>10</y>
      <width>561</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_count">
       <property name="text">
        <string>Количество ячеек</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox_count">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>200</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>240</y>
      <width>561</width>
      <height>131</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <widget class="QPushButton" name="pushButton_mid">
       <property name="text">
        <string>Среднее</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="pushButton_max">
       <property name="text">
        <string>Максимум</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="pushButton_min">
       <property name="text">
        <string>Минимум</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_min">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_mid">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label_max">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>410</y>
      <width>561</width>
      <height>251</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="pushButton_Comb">
       <property name="text">
        <string>Расчёстка</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Bubble">
       <property name="text">
        <string>Пузырёк</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Fast">
       <property name="text">
        <string>Быстрая </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_MONKE">
       <property name="text">
        <string>Обезьянья</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_GNOMED">
       <property name="text">
        <string>Гномья</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>370</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Сортировки:</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>660</y>
      <width>561</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_searching">
       <property name="text">
        <string>Поиск:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_searchValue"/>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_find">
       <property name="text">
        <string>найти</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>740</y>
      <width>561</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_searching_count">
       <property name="text">
        <string>Кол-во результатов:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_searching_countValue">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_searching_items">
       <property name="text">
        <string>Номера строк:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEdit_searching_itemsStroke"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>815</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>tableWidget_massive</tabstop>
  <tabstop>spinBox_count</tabstop>
  <tabstop>pushButton_random</tabstop>
  <tabstop>pushButton_removelast</tabstop>
  <tabstop>pushButton_randomValues</tabstop>
  <tabstop>pushButton_min</tabstop>
  <tabstop>pushButton_mid</tabstop>
  <tabstop>pushButton_max</tabstop>
  <tabstop>pushButton_Comb</tabstop>
  <tabstop>pushButton_Bubble</tabstop>
  <tabstop>pushButton_Fast</tabstop>
  <tabstop>pushButton_MONKE</tabstop>
  <tabstop>pushButton_GNOMED</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
